consultation_analyser/processing.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
```

Comment:
Shall we put these files in a namespace/app? Maybe a `pipeline` namespace?

Reply:
Now complete.